<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1152</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabs">
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="tabBarAutoHide">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="model_tab">
      <attribute name="title">
       <string>Setup</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QFrame" name="frame_4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Settings name:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="name_edit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QFrame" name="frame_5">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QGroupBox" name="category_box">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="title">
             <string>Model Category</string>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <property name="leftMargin">
              <number>20</number>
             </property>
             <property name="topMargin">
              <number>20</number>
             </property>
             <property name="rightMargin">
              <number>20</number>
             </property>
             <property name="bottomMargin">
              <number>20</number>
             </property>
             <item row="0" column="0">
              <widget class="QFrame" name="frame">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <item>
                 <widget class="QListWidget" name="model_category_list">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="currentRow">
                   <number>0</number>
                  </property>
                  <item>
                   <property name="text">
                    <string>Cubic</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>CPA</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="eos_box">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="title">
             <string>Equation of State</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <property name="leftMargin">
              <number>20</number>
             </property>
             <property name="topMargin">
              <number>20</number>
             </property>
             <property name="rightMargin">
              <number>20</number>
             </property>
             <property name="bottomMargin">
              <number>20</number>
             </property>
             <item row="0" column="0">
              <widget class="QStackedWidget" name="eos_stack">
               <property name="currentIndex">
                <number>1</number>
               </property>
               <widget class="QWidget" name="eos_page_0">
                <layout class="QVBoxLayout" name="verticalLayout_3">
                 <item>
                  <widget class="QListView" name="listView">
                   <property name="enabled">
                    <bool>false</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
               <widget class="QWidget" name="eos_page_1">
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <item>
                  <widget class="QListWidget" name="cubic_eos_list">
                   <property name="enabled">
                    <bool>true</bool>
                   </property>
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="currentRow">
                    <number>0</number>
                   </property>
                   <item>
                    <property name="text">
                     <string>PR</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>SRK</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                </layout>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="options_box">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="title">
             <string>Options</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QStackedWidget" name="options_stack">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="currentIndex">
                <number>1</number>
               </property>
               <widget class="QWidget" name="options_page_0"/>
               <widget class="QWidget" name="options_page_1">
                <layout class="QHBoxLayout" name="horizontalLayout_4">
                 <item>
                  <widget class="QFrame" name="frame_2">
                   <property name="frameShape">
                    <enum>QFrame::StyledPanel</enum>
                   </property>
                   <property name="frameShadow">
                    <enum>QFrame::Raised</enum>
                   </property>
                   <layout class="QFormLayout" name="formLayout">
                    <property name="fieldGrowthPolicy">
                     <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
                    </property>
                    <property name="rowWrapPolicy">
                     <enum>QFormLayout::WrapAllRows</enum>
                    </property>
                    <property name="formAlignment">
                     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                    </property>
                    <item row="0" column="0">
                     <widget class="QLabel" name="label">
                      <property name="text">
                       <string>Alpha correlaction</string>
                      </property>
                     </widget>
                    </item>
                    <item row="0" column="1">
                     <widget class="QComboBox" name="cubic_alpha_corr">
                      <property name="layoutDirection">
                       <enum>Qt::LeftToRight</enum>
                      </property>
                      <property name="editable">
                       <bool>false</bool>
                      </property>
                      <property name="currentText">
                       <string>Classic</string>
                      </property>
                      <property name="currentIndex">
                       <number>0</number>
                      </property>
                      <property name="insertPolicy">
                       <enum>QComboBox::InsertAtBottom</enum>
                      </property>
                      <property name="sizeAdjustPolicy">
                       <enum>QComboBox::AdjustToMinimumContentsLength</enum>
                      </property>
                      <item>
                       <property name="text">
                        <string>Classic</string>
                       </property>
                      </item>
                      <item>
                       <property name="text">
                        <string>Twu</string>
                       </property>
                      </item>
                      <item>
                       <property name="text">
                        <string>MC</string>
                       </property>
                      </item>
                     </widget>
                    </item>
                    <item row="2" column="0">
                     <widget class="QLabel" name="label_2">
                      <property name="text">
                       <string>Mixing rules</string>
                      </property>
                     </widget>
                    </item>
                    <item row="2" column="1">
                     <widget class="QComboBox" name="cubic_mix_rule">
                      <property name="editable">
                       <bool>false</bool>
                      </property>
                      <property name="currentText">
                       <string>vdW</string>
                      </property>
                      <property name="currentIndex">
                       <number>0</number>
                      </property>
                      <property name="sizeAdjustPolicy">
                       <enum>QComboBox::AdjustToMinimumContentsLength</enum>
                      </property>
                      <item>
                       <property name="text">
                        <string>vdW</string>
                       </property>
                      </item>
                      <item>
                       <property name="text">
                        <string>HV1</string>
                       </property>
                      </item>
                      <item>
                       <property name="text">
                        <string>HV2</string>
                       </property>
                      </item>
                     </widget>
                    </item>
                    <item row="3" column="0" colspan="2">
                     <spacer name="verticalSpacer_2">
                      <property name="orientation">
                       <enum>Qt::Vertical</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>20</width>
                        <height>40</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                    <item row="4" column="0">
                     <widget class="QLabel" name="label_3">
                      <property name="text">
                       <string>Volume translation</string>
                      </property>
                     </widget>
                    </item>
                    <item row="4" column="1">
                     <widget class="QComboBox" name="cubic_vol_trans">
                      <property name="editable">
                       <bool>false</bool>
                      </property>
                      <property name="currentText">
                       <string>None</string>
                      </property>
                      <property name="currentIndex">
                       <number>0</number>
                      </property>
                      <property name="sizeAdjustPolicy">
                       <enum>QComboBox::AdjustToMinimumContentsLength</enum>
                      </property>
                      <item>
                       <property name="text">
                        <string>None</string>
                       </property>
                      </item>
                      <item>
                       <property name="text">
                        <string>Peneloux</string>
                       </property>
                      </item>
                     </widget>
                    </item>
                    <item row="5" column="0" colspan="2">
                     <spacer name="verticalSpacer_3">
                      <property name="orientation">
                       <enum>Qt::Vertical</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>20</width>
                        <height>40</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                    <item row="6" column="0">
                     <widget class="QLabel" name="label_4">
                      <property name="text">
                       <string>Reference</string>
                      </property>
                     </widget>
                    </item>
                    <item row="6" column="1">
                     <widget class="QComboBox" name="cubic_ref">
                      <property name="editable">
                       <bool>false</bool>
                      </property>
                      <property name="currentText">
                       <string>Default</string>
                      </property>
                      <property name="currentIndex">
                       <number>0</number>
                      </property>
                      <property name="sizeAdjustPolicy">
                       <enum>QComboBox::AdjustToMinimumContentsLength</enum>
                      </property>
                      <item>
                       <property name="text">
                        <string>Default</string>
                       </property>
                      </item>
                     </widget>
                    </item>
                    <item row="1" column="0" colspan="2">
                     <spacer name="verticalSpacer">
                      <property name="orientation">
                       <enum>Qt::Vertical</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>20</width>
                        <height>40</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item>
                  <widget class="QFrame" name="frame_3">
                   <property name="frameShape">
                    <enum>QFrame::StyledPanel</enum>
                   </property>
                   <property name="frameShadow">
                    <enum>QFrame::Raised</enum>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="binary_coeff_tab">
      <attribute name="title">
       <string>Binary coefficients</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_10">
       <item row="0" column="0">
        <widget class="QStackedWidget" name="coeff_stack">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="page"/>
         <widget class="QWidget" name="page_2"/>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
